Core/src/main/java/support/Job.java

```diff
@@ -2,20 +2,29 @@
 
 import java.io.File;
 import java.nio.file.Paths;
-import java.util.Objects;
 
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+/*
+ * last ModifiedBy : @kuntal1461
+ */
```

Comment:
Git already tracks commit history, including timestamps and authorship. There's no need for manual `Last modified:` comments.

```suggestion
```